deployment/grid/terraform/compute_plane/variables.tf

```diff
@@ -169,37 +161,33 @@ variable "state_table_config" {
 
 variable "aws_xray_daemon_version" {
   description = "version for the XRay daemon"
-  type = string
+  type        = string
 }
 
 variable "enable_private_subnet" {
   description = "enable private subnet"
-  type = bool
+  type        = bool
 }
 
 variable "grafana_configuration" {
   description = "this variable store the configuration for the grafana helm chart"
   type = object({
-    downloadDashboardsImage_tag = string
-    grafana_tag = string
-    initChownData_tag = string
-    sidecar_tag = string
+    # downloadDashboardsImage_tag = string
+    # grafana_tag                 = string
+    # initChownData_tag           = string
+    # sidecar_tag                 = string
     admin_password = string
-
-    # busybox
-
-
   })
 }
 
-variable "prometheus_configuration" {
-  description = "this variable store the configuration for the prometheus helm chart"
-  type = object({
-    node_exporter_tag = string
-    server_tag = string
-    alertmanager_tag = string
-    kube_state_metrics_tag = string
-    pushgateway_tag = string
-    configmap_reload_tag = string
-  })
-}
\ No newline at end of file
+# variable "prometheus_configuration" {
```

Comment:
If this variable not use  any nore ?
If yes, we could just delete it ?

Reply:
I intentionally left it as I just commented it out in this release. Also, another place holder for a change in vars for a future PR I have in the plan.